scripts/build-charts-ci.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+set -e
+
+# Install chartmuseum using the gofish package manager
+curl -fsSL https://raw.githubusercontent.com/fishworks/gofish/main/scripts/install.sh | bash
+gofish init
+gofish install chartmuseum
+
+# We need the helm push plugin to automatically package and push chart to our repo
+helm plugin install https://github.com/chartmuseum/helm-push.git
+
+# Systemd unit file to start chartmuseum in the runner
+cat <<EOF | sudo tee /etc/systemd/system/chartmuseum.service
+[Unit]
+Description=Helm Chartmuseum
+Documentation=https://chartmuseum.com/
+
+[Service]
+ExecStart=/usr/local/bin/chartmuseum \\
+ --debug \\
+ --port=8090 \\
+ --storage="local" \\
+ --storage-local-rootdir="/home/${USER}/chartstorage/"
+Restart=on-failure
+RestartSec=5
+
+[Install]
+WantedBy=multi-user.target
+EOF
+
+sudo systemctl daemon-reload
+sudo systemctl start chartmuseum
+
+# Waiting chartmuseum initialization
+sleep 5
+
+# Add our new ephemeral repo
+helm repo add chartmuseum http://localhost:8090
+helm repo update
+
+# Push all our charts to the repo
+# We force version 0.1.0 because we don't care about versionning
+# We only care about testing here
+cd chart/
+helm cm-push --version "0.1.0" epinio-installer/ chartmuseum
+helm cm-push --version "0.1.0" epinio/ chartmuseum
+helm cm-push --version "0.1.0" container-registry/ chartmuseum
+
+# Update epinio-installer's values.yaml file for using charts hosted in our repo
+sed -i -e 's|containerRegistryChart:.*|containerRegistryChart: "http://host.k3d.internal:8090/charts/container-registry-0.1.0.tgz"|'\
```

Comment:
I'm not sure this host is resolvable from within the installer job. I think `resolv.conf` is inherited all the way down to pods but I'm not sure this will work.

Reply:
I think it works because in a previous test, I forgot `/chart/` in the repo url and I got this:
https://github.com/juadk/epinio-helm-chart/runs/4598040631?check_suite_focus=true#step:11:134
And now with the good url repo, it passes: https://github.com/juadk/epinio-helm-chart/runs/4601243506?check_suite_focus=true